examples/lighting-app/nrf5/main/AppTask.cpp

```diff
@@ -34,6 +34,12 @@
 #include "FreeRTOS.h"
 
 #include <platform/CHIPDeviceLayer.h>
+#include <platform/OpenThread/OpenThreadUtils.h>
+#include <platform/ThreadStackManager.h>
+#include <platform/internal/DeviceNetworkInfo.h>
+#include <platform/nRF5/ThreadStackManagerImpl.h>
```

Comment:
I think we're eventually going to have to push this further down into the stack. Having apps/examples be aware of this level of detail feels not great...